src/torcontrol.cpp

```diff
@@ -662,26 +662,26 @@ void TorController::reconnect_cb(evutil_socket_t fd, short what, void *arg)
 }
 
 /****** Thread ********/
-struct event_base *base;
```

Comment:
Hmm, that doesnt really answer my question, thanks gcc...Is that trying to say that boost has some "base" object that we're shadowing? Either way lets make the new gBase static.